extension/src/extension.ts

```diff
@@ -227,6 +238,19 @@ export class Extension extends Disposable implements IExtension {
 
     registerExperimentCommands(this.experiments, this.internalCommands)
     registerPlotsCommands(this.plots, this.internalCommands)
+    this.internalCommands.registerExternalCommand(
```

Comment:
Since this command touches both plots and experiments, I wasn't sure where to put it. For now I left it in `extension.ts` but we could move it into one of the above functions or its own file 🤔